neptune/management/internal/utils.py

```diff
@@ -52,3 +54,60 @@ def normalize_project_name(name: str, workspace: Optional[str] = None):
     )
 
     return f"{extracted_workspace_name}/{extracted_project_name}"
+
+
+class ProjectKeyGenerator:
```

Comment:
I think we could add  link to [docs](https://docs.neptune.ai/api-reference/management#.create_project) with information about key format.

Reply:
I added some comments, check it